src/components/card/card.css

```diff
@@ -0,0 +1,144 @@
+
+.container {
+  display: flex;
```

Comment:
remove display flex , the card is only a container base on widht and hight , user will add another container to styled as they want